runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/collections/general.ts

```diff
@@ -22,6 +22,7 @@ const general: typeof englishGeneral = {
     'Process.Collection.Tile.MenuItem.Edit': 'Edytuj',
     'Process.Collection.Navigation.Processes.Label': 'Procesy',
     'Process.Collection.Navigation.Collections.Label': 'Zbiory',
+    'Process.Collection.Path.Home': 'Dom',
```

Comment:
ask on RPA Offtop - what's the best translation - i'm not sure neither about 'dom', nor 'strona główna'